Moolya.robot

```diff
@@ -0,0 +1,152 @@
+*** Settings ***
+Library          SeleniumLibrary
+Resource         C:\\Users\\Admin\\Documents\\GitHub\\Mentorship_Abhijit\\BrowserKeywords.robot
+Suite Setup      Open Initial Browser
+Suite Teardown   Close All Browsers
+
+# All Keywords and variables are imported from BrowserKeywords robot file
+
+# To run Press Cntrl+Shift+P  in Windows
+# To run Press cmd-shift-P  in Mac
+
+
+
+*** Test Cases ***
+
+#To Setup Window
+Browser Setup
```

Comment:
This is not required you are already setting up the browser in Suite Setup. This helps for every test executed. It works the same as Before Suite in TestNG.

Reply:
Ok will remove that